### PR TITLE
[backbeat]: bump image tag to 8.0.18

### PR DESCRIPTION
**Pull request by giacope**
**What does this PR do, and why do we need it?**

Bumps backbeat image tag to `8.0.18`

**Which issue does this PR fix?**

None.

**Special notes for your reviewers**:

I'm assuming this should target `development/1.0`, therefore, the chart versions have been left intact. Is this ok? @ssalaues @rahulreddy.

### CODE REVIEW

**Comment by giacomoguiulfo** [comment]
@bert-e wait

**Review by alexanderchan-scality — APPROVED** [review]

**Review by JianqinWang — APPROVED** [review]

**Review by ssalaues — APPROVED** [review]

**Comment by rahulreddy** [comment]
Let's hold off. I realized I need to check in one more change.

File: kubernetes/zenko/charts/backbeat/Chart.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 apiVersion: v1
-appVersion: "8.0.16"
+appVersion: "8.0.18"
 description: A Helm chart for Zenko Backbeat
 name: backbeat
 version: 1.0.0
```
Comment (ssalaues) [inline kubernetes/zenko/charts/backbeat/Chart.yaml]: Lets bump these all to 1.0.1 as well. We're going to release a patch version.

**Comment by ssalaues** [comment]
@rahulreddy I imagine we should rebuild this docker image?

**Comment by rahulreddy** [comment]
Yes we should since we finally merged the PRs for tuneables.

**Review by rahulreddy — APPROVED** [review]

**Comment by ssalaues** [comment]
@bert-e reset

**Pull request merged**